wikimedia/wikipedia-ios
# Add new header view for article and user talk pages

**Phabricator:** https://phabricator.wikimedia.org/T310273

### Notes
Adds article and user talk page header to the new talk pages view controller. This single new header supports all the configurations I'm aware of for both article and user talk pages. 

- Need to actually populate this with real data once the middle layer talk pages data controller object is constructed
- Once the color style guide is finalized (https://phabricator.wikimedia.org/T299793), need to update the coffee roll background color to use the appropriate theme element.

### Test Steps
1. On a staging build, access the new talk pages view controller.
2. In `TalkPageViewModel`, change the `// TODO` sample data in order to play with the different configuration permutations. See the Phabricator ticket for some possible states.
